alune/config.py

```diff
@@ -122,20 +125,25 @@ def get_traits(self) -> list[images.Trait]:
         """
         return self._config["traits"]
 
-    def get_auto_surrend(self) -> bool:
+    def get_auto_surrender(self) -> bool:
         """
         Get the surrender option the user wants.
 
         Returns:
             A Boolean that determines whether or not we should surrend when possible.
         """
-        return self._config["auto_surrend"]
+        return self._config["auto_surrender"]
 
-    def get_auto_surrend_random_delay(self) -> int:
+    def get_auto_surrender_random_delay(self) -> int:
         """
         Get the surrender delay
 
         Returns:
-            An Integer that represents the number of maximum seconds to randomly wait before surrendering when possible
-        """
-        return self._config["auto_surrend_random_delay"]
+            An random Integer between [1 and auto_surrender_random_delay]
+            Returns 0 if feature disabled or negative value.
+        """
+        return (
+            0
+            if self._config["auto_surrender_random_delay"] <= 0
```

Comment:
nit: I'm a bit fuzzy on my syntax here, will this account for `null` / not present? 🤔

Reply:
Hello you are right, I have made a small change in ``config.py`` to check if it's ``NoneType``
Also, I forgot to await the ``check_surrender_state()`` so I awaited it :)